src/split-panel/side.tsx

```diff
@@ -37,13 +38,21 @@ export function SplitPanelContentSide({
   return (
     <div
       {...baseProps}
-      className={clsx(baseProps.className, styles.drawer, styles['position-side'], testUtilStyles.root, {
-        [testUtilStyles['open-position-side']]: isOpen,
-        [styles['drawer-closed']]: !isOpen,
-        [styles['with-toolbar']]: isAppLayoutToolbarEnabled(),
-      })}
+      className={clsx(
+        baseProps.className,
+        styles.drawer,
+        styles['position-side'],
+        testUtilStyles.root,
+        sharedStyles['with-motion'],
+        {
+          [testUtilStyles['open-position-side']]: isOpen,
+          [styles['drawer-closed']]: !isOpen,
+          [styles['with-toolbar']]: isAppLayoutToolbarEnabled(),
+          [styles.refresh]: isRefresh,
+        }
+      )}
       style={{
-        width: isOpen && isRefresh ? cappedSize : undefined,
+        width: isOpen ? cappedSize : '0px',
```

Comment:
With this change, the width for classic becomes `0` too, but in fact the closed drawer has some width there.

<img width="455" alt="image" src="https://github.com/user-attachments/assets/67944761-7dcb-4937-bae8-815792804b70">


---

It looks working because of some CSS magic, but still looks very sus? Can we keep it unchanged for classic?